### DISCUSSION TITLE
Are there statistics available (number of user, nodes etc)

### DISCUSSION THREAD

**Discussion by Tycho-X** [pocketnetteam/pocketnet.core#681] [General]
Hello,

i am currently evaluating NOSTR and Bastyon. For NOSTR there is a statistics page but i cant find any for Bastyon. Is there any data available?

cheers......

**Comment by andyoknen** [comment]
Hi!

maybe https://pocketnetexplorer.azurewebsites.net/#/nodetable ?
and https://bastyon.com/blockexplorer/
